.gitlab-ci.yml

```diff
@@ -0,0 +1,36 @@
+include:
```

Comment:
General question: if we add a `.gitlab-ci.yml` file at the root of this repo, wouldn't that complicate future OSS contributions? Wondering if you had some sort of mitigation strategy for that.

Reply:
Yes and No. I think it depends on the commit strategy we have as likely we'd be making targeted contributions with a defined set of commits (ie. just the diffs we want to push) so I think we'd need to improve our `git` management strategy (for example when I've been committing to `kfp` I've been manually rebasing and squashing `git` commits so we push a very well known change set). 

So really if the diffs we push upstream don't include the `.gitlab-ci.yml` changes (or any other "internal" only for now/or forever) then in theory we can just keep those changes in our fork, wherever it lives in the repo.